SQL/PostgreSQL/include/Poco/SQL/PostgreSQL/StatementExecutor.h

```diff
@@ -59,6 +59,10 @@ class StatementExecutor
 		/// Binds the params - REQUIRED if the statement has input parameters/placeholders
 		/// Pointer and list elements must stay valid for the lifetime of the StatementExecutor!
 
+	void bindBulkParams(const InputParameterVector& anInputBulkParameterVector);
```

Comment:
we want meaningful and descriptive names, but let's not exaggerate, this could be called eg. `inputBulkParamVec`